source/blog/2016-12-09-notice-to-kubernetes-atomic-users.html.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Important Notice for Users of Kubernetes on Fedora Atomic'
+author: jberkus
+date: 2016-12-09 16:00:00 UTC
+tags: fedora, atomic, kubernetes
+published: true
+comments: true
+---
+
+**TL;DR**: If you are a production user of Kubernetes on Fedora Atomic Host, please
+don't rebase to Fedora Atomic 25 yet.
+
+One of the features of the Fedora Atomic Host 25 release was decoupling Kubernetes
+from the base ostree for Atomic (this is true of the current CentOS Atomic Host
+as well).  That is, Kubernetes is no longer in the base install, you need to
+add it in as system containers and/or an overlay.  This is a step forwards for
+Atomic because it means that users can continuously update Atomic, and update
+Kubernetes on a different schedule which works for their cluster.  Since Kubernetes
+gets released four times a year, this lets developers update to the latest
+version, and production users stay on their production version.
+
+However, when we released Fedora Atomic 25, the containerized install of wasn't
+quite ready due to some unexpected upstream issues and delays with the Fedora
+Container Build Service.  At the time, we expected those issues to be resolved
+within a few days.  Instead, some have taken longer than expected and are still
+unresolved or waiting on PR review.
+
+The Fedora Atomic team is hard at work on getting a solution out for Kubernetes
+users and expect to have one before the holidays.  If you are able to help with
+building or testing, please speak up on the Atomic Development mailing list; we
+could use your help.  If you can't help, wait for us to publish documentation
+of the new containerized Kubernetes before you rebase to 25.
+
+If you use Fedora Atomic, but do not use Kubernetes, this issue does not affect
```

Comment:
use Kubernetes from the built-in image -- people running it from containers already, such as from upstream containers, won't be affected